server/svix-server/src/v1/utils.rs

```diff
@@ -20,18 +20,59 @@ use crate::{
     error::{Error, HttpError, Result, ValidationErrorItem},
 };
 
-const fn default_limit() -> u64 {
-    50
+const fn default_limit() -> ListLimit {
+    ListLimit(50)
 }
 
+const PAGINATION_LIMIT_CAP_HARD: bool = true;
+const PAGINATION_LIMIT_CAP_LIMIT: u64 = 250;
+// TODO: Should probably use lazy_static and format! to make this instead of repeating the 250
+// figure at some point
+const PAGINATION_LIMIT_ERROR: &str = "Given limit must not exceed 250";
+
 #[derive(Debug, Deserialize, Validate)]
 pub struct Pagination<T: Validate> {
+    #[validate]
     #[serde(default = "default_limit")]
-    pub limit: u64,
+    pub limit: ListLimit,
     #[validate]
     pub iterator: Option<T>,
 }
 
+#[derive(Debug)]
+pub struct ListLimit(pub u64);
```

Comment:
ListLimit was not a good name (sorry), can you search/replace to make it PaginationLimit?